changed larger to longer

```diff
@@ -9,30 +9,29 @@ amount you learn.
 
 Anyone who needs to remember things in their daily life can benefit from
 Anki. Since it is content-agnostic and supports images, audio, videos
-and scientific markup (via LaTeX), the possibilities are endless. For
+and scientific markup, the possibilities are endless. For
 example:
 
-- learning a language
+- Learning a language
 
-- studying for medical and law exams
+- Studying for medical and law exams
 
-- memorizing people’s names and faces
+- Memorizing names and faces of people
 
-- brushing up on geography
+- Brushing up on geography
 
-- mastering long poems
+- Mastering long poems
 
-- even practising guitar chords!
+- Even practising guitar chords!
 
 There are two simple concepts behind Anki: **active recall testing** and
-**spaced repetition**. They are not known to most learners, despite having
-been written about in the scientific literature for many years.
-Understanding how they work will make you a more effective learner.
+**spaced repetition**. They are not known to most learners, despite being well-documented in scientific literature.
+Understanding how these concepts work will make you a more effective learner.
 
 ## Active Recall Testing
 
 _Active recall testing_ means being asked a question and trying to
-remember the answer. This is in contrast to _passive study_, where we
+remember the answer. This is in contrast to _passive study_, where we 
```

Comment:
```suggestion
remember the answer. This is in contrast to _passive study_, where we
```

Trailing spaces can have a different meaning in markdown; better not to change it.